CONTRIBUTING.rst

```diff
@@ -264,6 +284,19 @@ If you wish to test a specific branch using GitHub CI, this can be set in `.gith
     In order for a Pull Request to be allowed to merge to main development branch, this variable must match the latest tagged commit name on `Ouranosinc/xclim-testdata`.
     We suggest merging changed testing data first, tagging a new version of `xclim-testdata`, then re-running tests on your Pull Request at `Ouranosinc/xclim` with the newest tag.
 
+Running Tests Offline
```

Comment:
Could you also explain what pytest-socket does and why its necessary ?

Reply:
Can do. I'll add a paragraph that points to the `pytest-socket` docs and briefly explains why we need such a complicated pytest call (because `pytest-xdist` relies on socket communication to properly function).

Reply:
@huard Just to clarify since this since I maybe did a terrible job explaining, but this "offline" test is only for those who wish to test the xclim without internet *while they are connected to the internet*. True offline testing (i.e., while on a train, to take a very random example) should work without all this. 

I'll make this much clearer in the docs.